project/docker/Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM alpine
 MAINTAINER Benedict Lau <benedict.lau@groundupworks.com>
 
-RUN apk add --update-cache
-RUN apk add autoconf automake build-base libtool ncurses nodejs nodejs-npm python
+RUN apk add --no-cache nodejs
```

Comment:
I was thinking to do it the same way as the ipfs docker, using a secondary builder docker. See: https://github.com/vanmesh/p2p-apps-dockers/blob/master/go-ipfs/Dockerfile#L30

Thoughts?

Reply:
@darkdrgn2k tried the builder approach used in the ipfs docker, and `sbot` cannot find dependencies. While it can be hacked to work, it seems the method here is more applicable for `npm install` flow. Going ahead with this :)